tests/test_sets.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+
+from src.sets import unique_elements_in_list
+
+
+class UniqueElementsInListTest(unittest.TestCase):
+
+    def test_equality(self):
+        test_list = [1, 1, 2, 3, 3]
+        self.assertEqual(sorted(unique_elements_in_list(test_list)), [1, 2, 3])
+
+        second_test_list = [4, 4, 5, 3, 5]
+        self.assertEqual(sorted(unique_elements_in_list(second_test_list)), [3, 4, 5])
+
+    def test_second_equality(self):
+        test_list = [2, 2, 3, 4, 3]
+
+        def list_check(mait_list):
+            unique_elements_list = unique_elements_in_list(test_list)
+            for i in unique_elements_list:
+                if i in mait_list:
+                    return True
```

Comment:
You will return `True` if first element of `unique_elements_list` is in `mait_list`, so you wouldn't check all elements from `unique_elements_list`